lib/exonio/financial.rb

```diff
@@ -119,6 +119,8 @@ def nper(rate, pmt, pv, fv = 0, end_or_beginning = 0)
     #   Exonio.pmt(0.075/12, 12*15, 200_000) # ==> -1854.0247200054619
     #
     def pmt(rate, nper, pv, fv = 0, end_or_beginning = 0)
+      return -(pv - fv) / nper if rate.zero?
```

Comment:
First of all, thanks for noticing the issue and opening this PR.

I wasn't sure about the formula, so I decided to check the test cases you added using NumPy Financial, and I got a different output for the second test case.

Running it against NumPy's implementation of the [PMT](https://numpy.org/numpy-financial/latest/pmt.html#numpy_financial.pmt) function, the returned result was `-1500` instead of `-500`.

To double-check, I used this [online calculator](https://www.calculator.net/finance-calculator.html), and the results matched those from NumPy.

Looking at NumPy's source code, I saw that the formula actually adds `fv` to `pv` before inverting the sign.

```python
# When `rate` is equal to 0, `tmp` is equal to 1 and `fact` is equal to `nper`
return -(fv + pv * temp) / fact
```

You can take a look at the full source code [here](https://github.com/numpy/numpy-financial/blob/f1a43fabd7cc51105cd7ac60c55465e7117c1519/numpy_financial/_financial.py#L262).

I will gladly merge this PR if you can fix the formula.

Reply:
Wait, I'm confused. I must completely misunderstand the meaning of `fv` (future value), but I concur that Excel and Google Sheets also have the same behavior that you pointed to.

I will fix.

Reply:
Fixed. Thanks.